packages/orchestration/src/utils/registry.js

```diff
@@ -53,8 +53,8 @@ function toConnectionEntry(ibcInfo, name, chainInfo) {
   }
   const [channel] = transferChannels;
   const [channelFrom, channelTo] = fromChain1
-    ? [channel.chain_2, channel.chain_1]
-    : [channel.chain_1, channel.chain_2];
+    ? [channel.chain_1, channel.chain_2]
```

Comment:
in hindsight a unit test for the direction would have helped. not worthwhile now imo